pkg/framework/version_bundle.go

```diff
@@ -46,6 +46,25 @@ func GetVersionBundleVersion(params *VBVParams) (string, error) {
 	return output, nil
 }
 
+func GetAuthorities(params *VBVParams) ([]versionbundle.Authority, error) {
+	err := checkType(params.VType)
+	if err != nil {
+		return []versionbundle.Authority{}, microerror.Mask(err)
+	}
+	log.Printf("Tested version %q", params.VType)
```

Comment:
I would like to use our logger as we do everywhere. This here will not play in favour of our perception when we humans deal with logs.

Reply:
Done, PTAL.